.mergify.yml

```diff
@@ -17,6 +14,6 @@ pull_request_rules:
       - status-success=license/cla
       - label!=do-not-merge
     actions:
-      merge:
+      queue:
         method: squash
-        strict: true
+        strict: default
```

Comment:
Looking at the blog post I think this should use `name` instead of `strict`

```yaml
actions:
      queue:
        method: merge
        name: default
```

Reply:
Thanks, I missed that one. Fixed.